lib/scrabble.rb

```diff
@@ -1,18 +1,35 @@
 class Scrabble
+  attr_reader :word
+
+  def initialize
+    @word = nil
```

Comment:
Recommend not adding this as an instance variable without having a specific reason to do so. In `score` you are basically treating it like a local variable, so seems unnecessary.